lecture/nn/nn5_mlp.md

```diff
@@ -23,6 +23,24 @@ youtube:
 attachments:
   - link: "https://raw.githubusercontent.com/Artificial-Intelligence-HSBI-TDU/KI-Vorlesung/master/lecture/nn/files/NN5-MLP.pdf"
     name: "NN5-MLP.pdf"
+challenges: |
+    **Lineares MLP**
+
+    Gegeben sei ein MLP mit linearen Aktivierungsfunktionen, d.h. für jedes Neuron berechnet sich der 
+    Output durch die gewichtete Summe der Inputs: $ y = g(w^T x) $, wobei $ g(z) = z $ gilt, also $ y = w^T x $.
```

Comment:
```suggestion
    Output durch die gewichtete Summe der Inputs: $y = g(w^T x)$, wobei $g(z) = z$ gilt, also $y = w^T x$.
```

Reply:
Wenn die `$` mit Leerzeichen vom Mathe-Inhalt getrennt sind, funktioniert das Parsing nicht mehr richtig in Hugo ... Ich habe das lokal mit dem Vorschlag getestet, da geht es dann.